synapse/handlers/register.py

```diff
@@ -804,12 +708,3 @@ def _register_msisdn_threepid(self, user_id, threepid, bind_msisdn):
         yield self._auth_handler.add_threepid(
             user_id, threepid["medium"], threepid["address"], threepid["validated_at"]
         )
-
-        if bind_msisdn:
```

Comment:
So we're no longer allowing clients to bind via during registration?

Reply:
Yes, as of MSC2140 `bind_email` and `bind_msisdn` options have been removed: https://github.com/matrix-org/matrix-doc/pull/2140/files#diff-c03a26de5ac40fb532de19cb7fc2aaf7R107

Reply:
That sounds like something that should have a changelog entry and be in a separate PR

Reply:
https://github.com/matrix-org/synapse/pull/5964